### ISSUE TITLE
bat traits for indexing by Encyclopedia of Life

### ISSUE THREAD

**Issue by jhpoelen** [jhpoelen/bigbatdb#1]
Hey @jhammock @katjaschulz - 

Any suggestions for how to structure statements like those describe on the attached photo?  What data format would you prefer? 

Context is that Hernani and Ariadna of the Big Data DB (see https://github.com/jhpoelen/bigbatdb) are trying to figure out how to get their records on Bat dental formula's published so that they can be easily indexed by EOL. Apparently, this dental formula is a trait used a bunch by bat researchers. 

The attached example was extracted from

Fenton, B., & Barclay, R. M. R. (1980). Myotis lucifugus. Mammalian Species, 142, 1–8. https://doi.org/10.5281/zenodo.13505746 https://linker.bio/hash://md5/791d7aef792ee698134b54daddb21eb4 

As far as I can tell, there's a ton more trait data where that come from with highly motivated folks in the bat research community to help make this available. 

Curious to hear your thoughts . . . 

![bigbatdb](https://github.com/user-attachments/assets/ffe76d2d-fa9f-4c41-81fb-01acdbe7cb3f)

**Comment by jhpoelen** [comment]
@KatjaSchulz thanks for sharing your thoughts on how to encode "dental formula" traits. Just curious - would you recommend the data format of any particular trait dataset you've worked with? Or perhaps go with measurementOrFact as part of a darwin core occurrence. Just making sure to get the format that'd be the least path of resistance to include in EOL.

**Comment by KatjaSchulz** [comment]
Yes, we usually use the DarwinCore measurementOrFact extension to to represent trait data records for import to EOL. We do not yet have any dentition data in EOL. I would recommend the following representation to get the discussion started:

[measurementTypes](http://rs.tdwg.org/dwc/terms/measurementType):

- http://purl.obolibrary.org/obo/UBERON_0003450 - upper jaw incisor
- http://purl.obolibrary.org/obo/UBERON_0018621 - upper canine tooth
- http://purl.obolibrary.org/obo/UBERON_0016944 - upper premolar tooth
- http://purl.obolibrary.org/obo/UBERON_0003666 - upper jaw molar
- http://purl.obolibrary.org/obo/UBERON_0003451 - lower jaw incisor
- http://purl.obolibrary.org/obo/UBERON_0018622 - lower canine tooth
- http://purl.obolibrary.org/obo/UBERON_0016943 - lower premolar tooth
- http://purl.obolibrary.org/obo/UBERON_0003667 - lower jaw molar

[measurementValues](http://rs.tdwg.org/dwc/terms/measurementValue):
dental formula representation for a given tooth

[statisticalMethod](http://eol.org/schema/terms/statisticalMethod):
http://purl.obolibrary.org/obo/STATO_0000047 - count

[bibliographicCitation](http://purl.org/dc/terms/bibliographicCitation):
The work cited for the dental formula

[source](http://purl.org/dc/terms/source):
The doi link or any other link for the source of the dental formula information

[measurementRemarks](http://rs.tdwg.org/dwc/terms/measurementRemarks):
For anything else you would want to add, e.g., you could provide the entire dental formula in simple notation

**Comment by jhpoelen** [comment]
@KatjaSchulz thanks again for sharing your ideas on how to organize the dental formula for reuse by Encyclopedia of Life (and others).

**Comment by jhpoelen** [comment]
@ariadnamorales - I started a tabular example of what I think @KatjaSchulz and @jhammock would want to their EOL Traitbank.

You can find the example at: 

https://docs.google.com/spreadsheets/d/1gtENmce4tv2AJWduWrj6dKnDFE0O-tLNZVAG6BAn5r4/edit?usp=sharing

See also attached csv/xlsx copies as made available through https://github.com/jhpoelen/bigbatdb and their track.sh .

[bigbatdb.csv](https://github.com/user-attachments/files/17689554/bigbatdb.csv)
[bigbatdb.xlsx](https://github.com/user-attachments/files/17689556/bigbatdb.xlsx)

**Comment by jhpoelen** [comment]
hey @KatjaSchulz @jhammock - I've attempted to create a dwc-a for the dental formula traits, and it appears that DwC-A validator of GBIF likes it. 

Now, I wonder . . . does EOL like it as far as importing the traits into traitbank ? 

Can you see whether you can make any sense of the https://github.com/bigbatdb/bigbatdb and their zipped version at https://github.com/BigBatDB/bigbatdb/archive/refs/heads/main.zip ? 

![image](https://github.com/user-attachments/assets/4f359c95-6976-4771-8081-8096ddb70861)

**Comment by KatjaSchulz** [comment]
I'm surprised this validates as a Darwin Core Archive, but regardless all the information we need is there, and we could easily extract it. We would use the following fields from the bigbatdb file:

family_AsListedInPublication
scientificName_AsListedInPublication	
measurementType
measurementValue
statisticalMethod
voucherID
specimenLocality
bibliographicCitation
bibliographicCitation_DOI_ZenondoBatLit
measurementRemarks_fromOriginalRef	
persObs_dataCollectorID
